source/user-guide/harvesting/harvesting-simpleurl.rst

```diff
@@ -0,0 +1,43 @@
+.. _harvesting-simpleurl-services:
+
+Simple URL harvesting (opendata)
+################################
+
+This harvester connects to a remote server via a simple URL to retrieve metadata records. This allows harvesting opendata catalogs such as opendatasoft and ESRI.
+
+Adding a simple URL harvester
+`````````````````````````````
+
+- **Site** - Options about the remote site.
+
+    - *Name* - This is a short description of the remote site. It will be shown in the harvesting main page as the name for this instance of the harvester.
+    - *Service URL* - The URL of the server to be harvested. This can include pagination params like ``?start=0&rows=20``
+    - *loopElement* - Propery/element containing a list of the record entries. (Indicated as an absolute path from the document root.) eg. ``/datasets``
+    - *numberOfRecordPath* : Property indicating the total count of record entries. (Indicated as an absolute path from the document root.) eg. ``/nhits``
+    - *recordIdPath* : Property containing the record id. eg. ``datasetid``
+    - *pageFromParam* : Property indicating the first record item on the current "page" eg. ``start``
+    - *pageSizeParam* : Property indicating the number of records containned in the current "page" eg. ``rows``
+    - *toISOConversion* : Name of the conversion schema to use, which must be available as XSL on the GN instance. eg. ``OPENDATASOFT-to-ISO19115-3-2018``
```

Comment:
Should we put something here about how conversion schema can come from different schema, and how they should be named in that case? eg `schema:iso19115-3.2018:convert/fromJsonOpenDataSoft` refers to the `fromJsonOpenDataSoft` xsl in the `convert` folder for the i`so19115-3.2018` schema

Reply:
@fgravin aside from my comment above, I think this looks good. I can't add anything about ckan settings though as I'm still struggling to get that one to work!